src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
+@import '../utils/variables';
+
+@keyframes hoverTransform {
+  from {
+    transform: translateY(0);
+  }
+
+  to {
+    transform: translateY(-20px);
+  }
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-columns;
+  max-width: (($day-size + $gap-columns) * $columns);
```

Comment:
The `max-width` calculation for `.calendar` seems to be off by one `$gap-columns`. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the `max-width` should be calculated as follows: `max-width: (($day-size + $gap-columns) * $columns - $gap-columns);` to account for one less gap than the number of columns.